docs/README.md

```diff
@@ -1,34 +1,64 @@
-# Requirements
+# About
 
-- ruby
-- bundler
+This is the repository for the Obojobo Next documentation (View the current docs at https://ucfopen.github.io/Obojobo-Docs/).
 
-# Installation
+Our documentation is versioned and built off of Jekyll.
+
+# File Structure & Versioning
+
+The actual markdown content for the releases are versioned and located in the `/releases` directory. Since Jekyll does not support versioning natively our system of versioning is explained here. Here's an example of the project's file structure:
+
+```
+Obojobo-Docs/
+	...
+	docs/
+	releases/
+		v3.3.2/
+		v3.3.3/
+		...
+		latest/
+```
+
+`./docs/` contain the output files. `./releases/` contain the versioned source files.
+
+`./releases/latest/` is a copy of the files inside the most recent version (in this example, 3.3.3) except each file contains a redirect to the file in the most recent version. This creates a URL to the latest version of a file. In the example above, if a file `./releases/v3.3.3/some-page.md` exists then a redirect-only version of that file will be placed in `./releases/latest/some-page.md`. The result of this is that when the docs are built the URL `http://.../releases/latest/some-page` will redirect to `http://.../releases/v3.3.3/some-page`.
+
+> Note: `./releases/latest` only creates redirects for files in the newest version - it will not create a redirect for any files that only exist in an older version. For example, if `./releases/v3.3.2/example.md` exists but `./releases/v3.3.3/example.md` does not then no redirect will exist in `./releases/latest`.
+
+> The `./releases/latest` directory is updated when the docs are built or on the first run when developing. There are also rake commands to do this (see "Update the latest redirects" below).
+
+# Getting Started
+
+## Installing
 
 - Run `bundle install`
 
-# Edit and live reloading the docs in the browser
+### Requirements
 
-- Run `bundle exec rake dev`
+- Ruby
+- Bundler
+
+# Developing the docs
 
-# Running the HTML Proofer
+- Run `bundle exec rake dev`
+- Go to http://127.0.0.1:4000/Obojobo-Docs/ in your browser
 
-- Run `bundle exec rake test`
+Updating the files in the `./releases` directory should auto-update in your browser.
```

Comment:
The files in the ./releases directory will auto-update in your browser when they are modified.